sdcm/cluster.py

```diff
@@ -3241,6 +3250,16 @@ def get_datacenter_name_per_region(self, db_nodes=None):
 
         return datacenter_name_per_region
 
+    def get_rack_names(self, db_nodes=None):
```

Comment:
I think this is not going to work, if rack names in different dc's differ.

We should take racks per DC and select one from it

Reply:
the idea is that we take all of the rack names, and try to match them with the loader rack name

so you are saying it would be something like

eu-west-1 
0: eu-west-1-a
1: eu-west-1-b
2: eu-west-1-c

us-east-1
0: us-east-1-a
1: us-east-1-b


so mapping base on the rack variable, which is just a number, would be enough ?

Reply:
yes, even in unit test second dc uses `2a` while first `1a`.
But now I see you fixed it.